<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataFrame</class>
 <widget class="QFrame" name="DataFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <property name="lineWidth">
   <number>0</number>
  </property>
  <widget class="QTabWidget" name="dataTabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>655</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::West</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="usesScrollButtons">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tabFiles">
    <property name="styleSheet">
     <string notr="true">#tabFiles
{
	
}</string>
    </property>
    <attribute name="title">
     <string>灌注文件</string>
    </attribute>
    <widget class="QWidget" name="fileWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1075</width>
       <height>520</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#fileWidget
{
background-color: rgb(64, 64, 71);
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pOpenBut">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>555</y>
       <width>160</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pOpenBut
{
background-image: url(:/images/dOpenBut_Icon.png);
}
#pOpenBut:pressed
{
background-image: url(:/images/dOpenButDn_Icon.png);
}
#pOpenBut:focus{outline: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>160</width>
       <height>60</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pExPortBut">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>555</y>
       <width>160</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pExPortBut
{
background-image: url(:/images/dExPortBut_Icon.png);
}
#pExPortBut:pressed
{
background-image: url(:/images/dExPortButDn_Icon.png);
}
#pExPortBut:focus{outline: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>160</width>
       <height>60</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pRfshBut">
     <property name="geometry">
      <rect>
       <x>555</x>
       <y>555</y>
       <width>160</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pRfshBut
{
background-image: url(:/images/dRfshBut_Icon.png);
}
#pRfshBut:pressed
{
background-image: url(:/images/dRfshButDn_Icon.png);
}
#pRfshBut:focus{outline: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>160</width>
       <height>60</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pDelBut">
     <property name="geometry">
      <rect>
       <x>737</x>
       <y>555</y>
       <width>160</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pDelBut
{
background-image: url(:/images/dDelBut_Icon.png);
}
#pDelBut:pressed
{
background-image: url(:/images/dDelButDn_Icon.png);
}
#pDelBut:focus{outline: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>160</width>
       <height>60</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pDelAllBut">
     <property name="geometry">
      <rect>
       <x>915</x>
       <y>555</y>
       <width>160</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pDelAllBut
{
background-image: url(:/images/dDelAllBut_Icon.png);
}
#pDelAllBut:pressed
{
background-image: url(:/images/dDelAllButDn_Icon.png);
}
#pDelAllBut:focus{outline: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>160</width>
       <height>60</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabPress">
    <attribute name="title">
     <string>压力曲线</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabSpeed">
    <attribute name="title">
     <string>转速曲线</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabFlow">
    <attribute name="title">
     <string>流量曲线</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabPumTmp">
    <attribute name="title">
     <string>温度曲线</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabWatTmp">
    <attribute name="title">
     <string>水浴温度</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabDetail">
    <property name="styleSheet">
     <string notr="true">#tabDetail
{
background-color: rgb(64, 64, 71);
}</string>
    </property>
    <attribute name="title">
     <string>详细数据</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
